scripts/controllers/editor.js

```diff
@@ -14,6 +14,10 @@ SwaggerEditor.controller('EditorCtrl', function EditorCtrl($scope, $rootScope,
     }
   });
 
+/**
+ * Get Debounced On Ace Change.
+ * @constructor
```

Comment:
This not a constructor. Specify the return type value (function returns debounced unchanged function)